packages/rocketchat-livechat/client/lib/Analytics.js

```diff
@@ -0,0 +1,116 @@
+RocketChat.Livechat = {};
```

Comment:
What is the purpose of this? =)

Reply:
RocketChat.Livechat isn't available on client, so had to define.